gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -48,6 +48,13 @@ var gpii = fluid.registerNamespace("gpii");
         members: {
             sessionIndex: {} // map of userToken to session component member name, managed by gpii.lifecycleManager.sessionIndexer
         },
+        model: {
+            logonChangeInProgressFor: undefined, // will have the value of the user token
```

Comment:
The more I think about it, it seems a great opportunity to eliminate all your special "onLogonChangeStarted" etc. events and simply modelise this workflow with the following structure:
logonChange: {
    type: // "login" / "logout"
    inProgress: // true / false
    userToken: // string
    timestamp: // int
}
Then everything that wants to express an interest can simply listen to, e.g. logonChange.inProgress